Description: Use explicit dep: syntax to reduce test cases generated
Last-Update: 2025-08-16
Author: Blair Noctis <ncts@debian.org>
Forwarded: not-needed
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -57,3 +57,3 @@
-    "async-lock",
-    "event-listener",
-    "futures-util",
+    "dep:async-lock",
+    "dep:event-listener",
+    "dep:futures-util",
@@ -61 +61 @@
-logging = ["log"]
+logging = ["dep:log"]
@@ -196 +196 @@
-[dependencies.once_cell]
+[disabled.dependencies.once_cell]
